src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -231,16 +246,15 @@ export default class ChangeBusinessType extends Mixins(CommonMixin, EnumMixin) {
   // Global getters
   @Getter getApprovedName!: string
   @Getter getEntityType!: CorpTypeCd
-  @Getter isConflictingLegalType!: boolean
-  @Getter getBusinessSnapshot!: BusinessSnapshotIF
-
-  // Alteration flag getters
+  @Getter getEntitySnapshot!: EntitySnapshotIF
+  @Getter getResource!: ResourceIF
```

Comment:
Since this component uses global getter (esp resource) then I'd say this is a middle-level component, even though it's in a sub-sub-folder and uses no lower components. And that's fine; the architecture can be a bit flexible :)

Reply:
Good point, this is worth a closer look tho!